python/pyspark/sql.py

```diff
@@ -2284,6 +2284,18 @@ def addColumn(self, colName, col):
         """
         return self.select('*', col.alias(colName))
 
+    def to_pandas(self):
+        """
+        Collect all the rows and return a `pandas.DataFrame`.
+
+        >>> df.to_pandas()  # doctest: +SKIP
```

Comment:
should this be called `to_panda` since a single "panda" is equivalent to a data frame